### PR TITLE
LPD-25594 fixed missing phone number when posting account via api endpoint

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-25594

### CODE REVIEW

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java [hunk 1]
```diff
@@ -670,6 +639,35 @@ null, _getType(account), _getStatus(account),
 		return _toAccount(accountEntry);
 	}
 
+	private void _addAddresses(Long accountId, Account account)
+		throws Exception {
+
+		PostalAddress[] postalAddresses = account.getPostalAddresses();
+
+		if ((postalAddresses != null) && (postalAddresses.length > 0)) {
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java:647]: if (postalAddresses == null or postalAddresses.length == 0) {
  return Collections.emptyList();
} so we can unindent the following code

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java [hunk 2]
```diff
@@ -670,6 +639,35 @@ null, _getType(account), _getStatus(account),
...
+			List<PostalAddress> filteredPostalAddresses = ListUtil.filter(
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java:648]: inline this below

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:um

**Comment by brbalazs** [comment]
ci:report:2568390213

**Comment by brbalazs** [comment]
ci:report:2568409725

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**